.github/workflows/test-runner-resiliency-test.yml

```diff
@@ -85,3 +85,10 @@ jobs:
           cd workspace-manager-clienttests
           echo "Upload test results to Google Bucket"
           ./gradlew uploadResults --args="CompressDirectoryToTerraKernelK8S.json /tmp/TestRunnerResults"
+
```

Comment:
GitHub won't let me comment on the above line, but I don't think this GH action will work anymore because it's overriding the server specification with a file that you deleted.

Reply:
Fixed the file to `workspace-wsmtest.json`, the workflow worked now.